### ISSUE TITLE
Keep getting kicked after every game?

### ISSUE THREAD

**Issue by WolfNoise** [fiskybusiness/phasmophobia#21]
After every game I play, whether I'm the host or not I always get kicked. Not sure if there's a specific setting that's doing it, but it consistently happens.

**Comment by Toonsie** [comment]
Same issue happening with me. Not the biggest problem, simple to rejoin and still get rewards but if possible to be solved, would be great!

**Comment by myrukun** [comment]
yeah this happens to me all the time. I either just get kicked or sent to a different lobby

**Comment by myrukun** [comment]
bump, this is really annoying. I might consider deleting this mod if it keeps happening. I find a super good lobby with all mics, we complete a contract and I lose them.
